schemas/eq_submission_schema.json

```diff
@@ -187,7 +190,27 @@
       "title": "Questionnaire ID",
       "default": "",
       "examples": [
-        "4782576236675773"
+        "7539657418968524"
+      ],
+      "pattern": "^(.*)$"
+    },
+    "case_type": {
+      "$id": "#/properties/case_type",
+      "type": "string",
+      "title": "The Case_type Schema",
+      "default": "",
+      "examples": [
+        "HI"
+      ],
+      "pattern": "^(.*)$"
+    },
+    "region_code": {
+      "$id": "#/properties/region_code",
+      "type": "string",
```

Comment:
Would be useful to add a description showing that this should be `A region code defined as per ISO 3166-2:GB`

Reply:
You could even include the regex from runner if you felt like it